src/Domain/BaseLiner/BaseLineImporter.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+/**
+ * Static Analysis Results Baseliner (sarb).
+ *
+ * (c) Dave Liddament
+ *
+ * For the full copyright and licence information please view the LICENSE file distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace DaveLiddament\StaticAnalysisResultsBaseliner\Domain\BaseLiner;
+
+use DaveLiddament\StaticAnalysisResultsBaseliner\Domain\Common\BaseLine;
+use DaveLiddament\StaticAnalysisResultsBaseliner\Domain\Common\FileName;
+use DaveLiddament\StaticAnalysisResultsBaseliner\Domain\File\FileAccessException;
+use DaveLiddament\StaticAnalysisResultsBaseliner\Domain\File\FileImportException;
+use DaveLiddament\StaticAnalysisResultsBaseliner\Domain\File\FileReader;
+use DaveLiddament\StaticAnalysisResultsBaseliner\Domain\HistoryAnalyser\HistoryFactoryLookupService;
+use DaveLiddament\StaticAnalysisResultsBaseliner\Domain\HistoryAnalyser\InvalidHistoryFactoryException;
+use DaveLiddament\StaticAnalysisResultsBaseliner\Domain\ResultsParser\AnalysisResults;
+use DaveLiddament\StaticAnalysisResultsBaseliner\Domain\ResultsParser\InvalidResultsParserException;
+use DaveLiddament\StaticAnalysisResultsBaseliner\Domain\ResultsParser\ResultsParserLookupService;
+use DaveLiddament\StaticAnalysisResultsBaseliner\Domain\Utils\ArrayParseException;
```

Comment:
I expect array parsing isn't really part of this domain. Maybe bump those `Utils` up a folder?

`DaveLiddament\StaticAnalysisResultsBaseliner\Utils\ArrayParseException`

Reply:
Thanks for comment. 
Yes. I think I might have named `Domain` incorrectly. I'll raise as an issue so we can talk through options further and come up with a better name. See: https://github.com/DaveLiddament/sarb/issues/4